### ISSUE TITLE
Question: Is it working with Muse 2

### ISSUE THREAD

**Issue by pwittchen** [tomov/muse-tracking-app#1]
Hi,

I have a question: Does your app work with the Muse 2 device? I heard that the latest hardware have problems with working with the old Muse SDK. Do you use old SDK or something else?

Regards,
Piotr

**Comment by tomov** [comment]
Yes it works with whatever Muse is the latest around December. You might notice that the code is in swift but actually uses methods from an objective C class. Let me know if you end up having any issues

Momchil Tomov

**Comment by pwittchen** [comment]
Thanks for the reply :)
